dashboard/app/views/layouts/_analytics.html.haml

```diff
@@ -25,21 +25,9 @@
 
   ga('set', dimensions);
   ga('send', 'pageview');
-  var userTimings = {};
   function trackEvent(category, action, label, value) {
     ga('send', 'event', category, action, label, value);
   }
-  function startTiming(category, variable, label) {
-    var key = category + variable + label;
-    userTimings[key] = new Date().getTime();
-  }
-  function stopTiming(category, variable, label) {
-    var key = category + variable + label;
-    var endTime = new Date().getTime();
-    var startTime = userTimings[key];
-    var timeElapsed = endTime - startTime;
-    ga('send', 'timing', category, variable, timeElapsed, label);
-  }
```

Comment:
These were only being used in initApp.js, so I moved them to a new file that lives alongside that one.